jdaviz/configs/rampviz/plugins/parsers.py

```diff
@@ -133,6 +152,13 @@ def parse_data(app, file_obj, data_type=None, data_label=None,
         raise NotImplementedError(f'Unsupported data format: {file_obj}')
 
 
+def _swap_axes(x):
+    # swap axes per the conventions of ramp cubes
+    # (group axis comes first) and the default in
+    # rampviz (group axis expected last)
+    return np.swapaxes(x, 0, -1)
```

Comment:
Do you ever plan to support beyond Roman vs JWST? Maybe instead of swapping, should hardcode the index by telescope name?

Reply:
It's not in the plans for now. If they become part of the plan, we'd need to build a parser for those "non-standard" cubes anyway, so I'll keep it like this for now.